packages/atoms/src/components/kiwi-label/kiwi-label.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { KiwiLabel, LabelKind } from './kiwi-label';
+import { toPairs } from 'lodash-es';
+import { newSpecPage } from '@stencil/core/testing';
+
+const testCases: { [Key in LabelKind]: Key } = {
+  danger: 'danger',
+  default: 'default',
+  info: 'info',
+  success: 'success',
+  warning: 'warning',
+};
+
+describe('kiwi-label', () => {
+  it('builds', () => {
```

Comment:
Willst du diese `it('builds')` tests gleich noch raushauen? Mir war so als ist das nicht mehr _state of the art_ ;) Und viel Mehrwert hats ja eh nicht.

Reply:
Kann ich machen.